homeworks/evgeny.tolchinsky_ryvik13/5-logic/homework.js

```diff
@@ -0,0 +1,37 @@
+/**
+ *
+ * @param firstValue string with first number
+ * @param secondValue string with second number
+ * @param operation string with operation symbol
+ * @returns {string|number} result of calculation or error message:
+ * - if firstValue or secondValue is not a number, return 'Enter a number'
+ * - if operation is not '+', '-', '*', '/', return 'Choose a valid operation'
+ * - if the result is more than 100, return 'Result is too big'
+ * - else return number of result: 1 '+' 2 returns 3
+ *
+ */
+function calculate(firstValue, secondValue, operation) {
+    const Number1 = Number(firstValue);
+    const Number2 = Number(secondValue);
+
+    if (!Number1 || !Number2) {
+        return 'Enter a number';
+    }
```

Comment:
Please add an error when a user tries to divide by 0. However, in all other cases when the second number is 0, the calculator should work properly.